packages/publikator/graphql/schema-types.js

```diff
@@ -69,13 +69,20 @@ type RepoMeta {
   briefingUrl: String
 }
 
+input RepoDateRange {
```

Comment:
Now that I see it, could we harden naming? Going for specific date, so `RepoPublishDateRange` and `publishDateRange` throughout?

Reply:
sure can do :)